core/src/main/resources/hudson/PluginManager/_table.js

```diff
@@ -5,6 +5,9 @@ function checkPluginsWithoutWarnings() {
         if(candidate.type === "checkbox" && candidate.dataset.compatWarning === 'false') {
             candidate.checked = true;
         }
+        else if(inputs[i].type === "checkbox") {
```

Comment:
`inputs[i]` == `candidate` so why are we not using the nicer local var here? Especially since with the next line this is really confusing code.

Also, this can probably be simplified to:

         if(candidate.type === "checkbox") {
            candidate.checked = candidate.dataset.compatWarning === 'false';
        }

Reply:
You are right, should be using `candidate` instead.

That said, I really like your proposed alternative. Very elegant. Will use it :)

Reply:
Done 👍